### ISSUE TITLE
IE 11 Error "Expected ':'"

### ISSUE THREAD

**Issue by LuXDAmore** [nuxt/components#56]
**Describe the bug**
Broken website after upgrading from `"nuxt": "^2.12.2"` to `"nuxt": "^2.13.0"` on IE 11.

**To Reproduce**
1. `nuxt: v2.13.0`;
2. `npm run dev`;
3. Open the console in a IE 11 browser.

**Expected behavior**
No errors.

**Screenshots**
![IE 11 error](https://user-images.githubusercontent.com/3841171/85135951-34fe8280-b23f-11ea-933a-93cdae39ad10.png)

**Additional context**
The error was not present before the upgrade.

**Comment by LuXDAmore** [comment]
Production build, **Syntax Error**:
![IE 11 - Production](https://user-images.githubusercontent.com/3841171/85142726-1604ee00-b249-11ea-87a6-00cca373a7d4.png)

I can see an arrow function `(t,a,b)=>` related to:

```javascript

  component.exports.render = function (h, vm) {
    return render(h, Object.assign({}, vm, {
      _c(n, a, b) {
        return vm._c(components[n] || n, a, b)
      }
    }))
  }

```

**Comment by LuXDAmore** [comment]
Setting `components: false` into the `nuxt.config.js` doesn't fix the problem, neither `features: { components: false }`.

(It's not clear for me if it's a feature or something else)

**Comment by pi0** [comment]
Hi @LuXDAmore. It is fixed with v1.0.3 (you can use `yarn upgrade nuxt`) 

> Setting components: false into the nuxt.config.js doesn't fix the problem

It is strange indeed because with `components: false` components module will be disabled unless manually specified from `modules` or `buildModules`

**Comment by LuXDAmore** [comment]
@pi0 Thank you very much!  🚀 

The issue is fixed now with `yarn upgrade nuxt`.. But yes, still persist on the `previous version` with `components: false` and nothing specified in `modules` or `buildModules`!

Hope this helps!
